package.json

```diff
@@ -201,11 +201,16 @@
                             "request": "launch",
                             "program": "^\"\\${command:ask.debugAdapterPath}\"",
                             "args": [
-                                "--accessToken", "^\"\\${command:ask.accessToken}\"",
-                                "--skillId", "^\"\\${command:ask.skillIdFromWorkspace}\"",
-                                "--handlerName", "handler",
-                                "--skillEntryFile", "^\"\\${workspaceFolder}/lambda/index.js\"",
-                                "--region", "${1|NA,EU,FE|}"
+                                "--accessToken",
```

Comment:
Why are we reformatting this? I kinda like the previous way of the key and value being in the same line

Reply:
once you run `npm install`, it seems package.json will be formatted automatically like this

Reply:
Exactly. Not in my hands :-(